src/OnboardMediaControl/OnboardMediaControl.pro

```diff
@@ -1,17 +1,21 @@
 TEMPLATE = app
-LIBS += -L../ViewLayer/.lib -L../BusinessLayer/.lib -lViewLayer -lBusinessLayer
+LIBS += -L../ViewLayer/.lib -L../BusinessLayer/.lib -lViewLayer -lBusinessLayer -L../External/.lib -lExternal
```

Comment:
For consistency:
`LIBS += -L../ViewLayer/.lib -L../BusinessLayer/.lib -L../External/.lib -lViewLayer -lBusinessLayer -lExternal`

Reply:
Okay